analyze.go

```diff
@@ -2074,6 +2074,9 @@ var directives = map[string][]uint{
 	"auth_jwt_type": {
 		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLmtConf | ngxConfTake1,
 	},
+	"auth_jwt_require": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLmtConf | ngxConfTake1,
```

Comment:
Can you check and see how many parameters you can use with `auth_jwt_require`? From the docs, it looks like more than one.

Reply:
you're right, it takes more than 1 args. so I changed to `ngxConf1More,`